src/Microsoft.AspNetCore.Mvc.Core/ApiConventionAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.Core;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    [AttributeUsage(AttributeTargets.Assembly | AttributeTargets.Class, AllowMultiple = true, Inherited = true)]
+    public sealed class ApiConventionAttribute : Attribute, IFilterMetadata
```

Comment:
We need a different way of specifying metadata(not specific to MVC) rather than using IFilterMetadata. I am thinking from perspective of Dispatcher, endpoints having metadata etc. No action needed on this PR though. cc @rynowak

Reply:
My idea for this is that we will map all of MVC's filters to dispatcher metadata.